core/coreapi/interface/interface.go

```diff
@@ -55,6 +58,37 @@ type UnixfsAPI interface {
 	Ls(context.Context, Path) ([]*Link, error)
 }
 
+// DagAPI specifies the interface to IPLD
+type DagAPI interface {
+	// Put inserts data using specified format and input encoding.
+	// If format is not specified (nil), default dag-cbor/sha256 is used
```

Comment:
`Unless used with WithCodec or WithHash, the defaults "dag-cbor" and "sha256" are used.`